source/linux/Foundational_Components/Kernel/_Users_Guide.rst

```diff
@@ -515,85 +515,168 @@ Describing FIT source
+               sign-images = "kernel", "fdt";
             };
-    };
+         };
 
 
-Specify all images you need the signature to authenticate as a part of
-sign-images. The key-name-hint needs to be changed if you are using some
-other key other than the TI dummy key that we are using for this example.
-It should be the name of the file containing the keys.
+   Specify all images you need the signature to authenticate as a part of
+   sign-images. The key-name-hint needs to be changed if you are using some
+   other key other than the TI dummy key that we are using for this example.
+   It should be the name of the file containing the keys.
 
-.. note::
+   .. note::
 
-    Generating new set of keys:
+      Generating new set of keys:
 
-    .. code-block:: console
+         .. code-block:: console
 
-        $ mkdir keys
-        $ openssl genpkey -algorithm RSA -out keys/dev.key \
-        -pkeyopt rsa_keygen_bits:4096 -pkeyopt rsa_keygen_pubexp:65537
-        $ openssl req -batch -new -x509 -key keys/dev.key -out keys/dev.crt
+            $ mkdir keys
+            $ openssl genpkey -algorithm RSA -out keys/dev.key \
+             -pkeyopt rsa_keygen_bits:4096 -pkeyopt rsa_keygen_pubexp:65537
+            $ openssl req -batch -new -x509 -key keys/dev.key -out keys/dev.crt
 
 Generating the fitImage
 ^^^^^^^^^^^^^^^^^^^^^^^
 
+.. ifconfig:: CONFIG_part_family not in ('General_family', 'AM335X_family', 'AM437X_family', 'AM57X_family')
+
+   .. note::
+
+      For signing a secondary platform like SK boards, you'll require
+      additional steps
+
+      - Change the CONFIG_DEFAULT_DEVICE_TREE
+
+       For e.g
+
+       .. code-block:: diff
+
+          diff --git a/configs/j721e_evm_a72_defconfig b/configs/j721e_evm_a72_defconfig
+	  index a5c1df7e0054..6d0126d955ef 100644
+	  --- a/configs/j721e_evm_a72_defconfig
+	  +++ b/configs/j721e_evm_a72_defconfig
+	  @@ -13,7 +13,7 @@ CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0x80480000
+	  CONFIG_ENV_SIZE=0x20000
+	  CONFIG_DM_GPIO=y
+	  CONFIG_SPL_DM_SPI=y
+	  -CONFIG_DEFAULT_DEVICE_TREE="k3-j721e-common-proc-board"
+	  +CONFIG_DEFAULT_DEVICE_TREE="k3-j721e-sk"
+	  CONFIG_SPL_TEXT_BASE=0x80080000
+	  CONFIG_DM_RESET=y
+	  CONFIG_SPL_MMC=y
+
+      - Change the binman nodes to package u-boot.dtb for the correct set of platform
+
+      For e.g
+
+      .. code-block:: diff
+
+         diff --git a/arch/arm/dts/k3-j721e-binman.dtsi b/arch/arm/dts/k3-j721e-binman.dtsi
+         index 673be646b1e3..752fa805fe8d 100644
+         --- a/arch/arm/dts/k3-j721e-binman.dtsi
+         +++ b/arch/arm/dts/k3-j721e-binman.dtsi
+         @@ -299,8 +299,8 @@
+         #define SPL_J721E_SK_DTB "spl/dts/k3-j721e-sk.dtb"
+
+         #define UBOOT_NODTB "u-boot-nodtb.bin"
+         -#define J721E_EVM_DTB "u-boot.dtb"
+         -#define J721E_SK_DTB "arch/arm/dts/k3-j721e-sk.dtb"
+         +#define J721E_EVM_DTB "arch/arm/dts/k3-j721e-common-proc-board.dtb"
+         +#define J721E_SK_DTB "u-boot.dtb"
+
```

Comment:
This seems like it should be indented to fall under the list item directly above it